seriously-simple-podcasting.php

```diff
@@ -29,16 +29,18 @@
 use SeriouslySimplePodcasting\Controllers\Settings_Controller;
 use SeriouslySimplePodcasting\Controllers\Options_Controller;
 use SeriouslySimplePodcasting\Rest\Rest_Api_Controller;
+use SeriouslySimplePodcasting\Controllers\Players_Controller;
+use SeriouslySimplePodcasting\Integrations\Elementor\Elementor_Widgets;
 
-define( 'SSP_VERSION', '2.2.3' );
+define( 'SSP_VERSION', '2.2.4-dev' );
 define( 'SSP_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
 define( 'SSP_PLUGIN_PATH', plugin_dir_path( __FILE__ ) );
 
 if ( ! defined( 'SSP_CASTOS_APP_URL' ) ) {
-	define( 'SSP_CASTOS_APP_URL', 'https://app.castos.com/' );
```

Comment:
Please change these constants back, and rather set them in your local wp-config.php